styles/styles.css

```diff
@@ -1579,4 +1580,16 @@ main .section.full-width > div {
   }
 }
 
+@media screen and (min-width: 1920px) {
+  main .section.column-with-list.blue-background.columns-container,
+  main .section.faq-container:not(.form-container, .plain)  {
```

Comment:
I feel like this is the wrong way to handle this. Seems more like a last minute work around than actual fix. We should not handle section styles dependant on it's block; that's a big no no. We should look for the reason it's ignoring the `max-width` of the section in the first place and get rid of that.